reports/guild-count.js

```diff
@@ -0,0 +1,20 @@
+// How many Guilds are users members of? Outputs counts of users with each number of Guilds
+
+var aggregationCursor = db.users.aggregate(
```

Comment:
Yeah, the lint tests are actually failing, I think it was included by default >.< I'll see if it'll run after cleaning up the linting.